server/rpc.go

```diff
@@ -367,9 +367,29 @@ func getOrchestrator(orch Orchestrator, req *net.OrchestratorRequest) (*net.Orch
 		return orchestratorInfo(orch, addr, orch.ServiceURI().String(), "")
 	}
 
+	if err := checkLiveVideoToVideoCapacity(orch, req.Capabilities); err != nil {
+		return nil, fmt.Errorf("Invalid orchestrator request: %v", err)
+	}
 	return orchestratorInfoWithCaps(orch, addr, orch.ServiceURI().String(), "", req.Capabilities)
 }
 
+func checkLiveVideoToVideoCapacity(orch Orchestrator, caps *net.Capabilities) interface{} {
+	if caps.Constraints != nil && caps.Constraints.PerCapability != nil {
+		if liveCap, ok := caps.Constraints.PerCapability[uint32(core.Capability_LiveVideoToVideo)]; ok {
+			pipeline := "live-video-to-video"
+			for modelID := range liveCap.GetModels() {
+				if orch.CheckAICapacity(pipeline, modelID) {
+					// It has capacity for at least one of the requested models
+					return nil
+				}
+			}
+			// No capacity for any requested model
+			return core.ErrOrchCap
+		}
+	}
+	return nil
```

Comment:
Should this default to return `core.ErrOrchCap`? 

If we get here then the Orch has no constraints or is missing the `live-video-to-video` capability so i think should assume cannot process request.

Reply:
I wanted to keep the non-live-video-to-video pipelines to process as before. My understanding is that for AI Jobs, we don't want to check capacity, because the jobs are short, so the selection can take into considerations busy Os. It's different for Live Video to Video.